roxie/ccd/ccdqueue.cpp

```diff
@@ -1369,6 +1369,15 @@ class CRoxieWorker : public CInterface, implements IPooledThread
                     slavesActive--;
                 }
             }
+            catch(IException *E)
+            {
+                CriticalBlock b(actCrit);
+                EXCLOG(E);
+                if (packet)
+                    throwRemoteException(E, NULL, packet, false);
+                packet.clear();
```

Comment:
Should release E (and probably shouldn't need to clear packet)

Reply:
Changed